src/components/Nav/Nav.js

```diff
@@ -1,7 +1,90 @@
 import React from 'react';
+import { withRouter } from 'react-router';
 
-function Nav() {
-  return <div>Nav</div>;
+import styled from 'styled-components';
+import SubNav from './SubNav/SubNav';
+
+function Nav(props) {
+  console.log(props);
```

Comment:
```suggestion
function Nav({ location }) {
```
- console.log...
- 함수 컴포넌트 자리에서 직접 구조분해할당 가능합니다